src/modules/components/Dashboard/Cards/Rewards/QueueCard.tsx

```diff
@@ -6,14 +6,25 @@ import CardTitle from '../CardTitle'
 import DashboardCard from '../DashboardCard'
 import { TrophyIcon } from './TrophyIcon'
 
+import { useMetrics } from '@/hooks/useApiGogopool'
 import { ordinal_suffix } from '@/utils/misc'
 
 export type Props = {
   minipoolIndex: number
-  nextCycleDate: Date
 }
 
-export default function QueueCard({ minipoolIndex, nextCycleDate }: Props) {
+export default function QueueCard({ minipoolIndex }: Props) {
+  const { data: metrics } = useMetrics()
+
+  let date = 'Loading...'
+
+  if (metrics) {
+    const cycleStart = metrics?.rewards_cycle_start_time
```

Comment:
nit: what you really care about in this block is the `rewards_cycle_start_time`, otherwise the date will be malformed.  might as well check that in order to prove the `cycleStart` won't be undefined when rendering the date.

```suggestion
  if (metrics?.rewards_cycle_start_time) {
    const cycleStart = metrics.rewards_cycle_start_time
```